src/cli/test_command.zig

```diff
@@ -719,11 +719,28 @@ pub const TestCommand = struct {
             // Treat arguments as filters and scan the codebase
             const filter_names = if (ctx.positionals.len == 0) &[0][]const u8{} else ctx.positionals[1..];
 
+            const filter_names_normalized = if (!Environment.isWindows)
+                filter_names
+            else brk: {
+                const normalized = try ctx.allocator.alloc([]const u8, filter_names.len);
```

Comment:
I might need a similar thing for bun install positionals. I wonder how hard it would be to have clap handle this